src/main/java/com/simongineer/diffuse_match/utils/Local.java

```diff
@@ -70,8 +75,9 @@ private static String printFile(String fileName) {
      * 
      * @param imageBytes The base64 image data to save.
      */
-    public static void saveGeneratedImage(byte[] imageBytes, Prompt prompt) {
-        File pngFile = new File("pics/" + prompt.getPrompt() + "_"
+    public static void saveGeneratedImage(byte[] imageBytes, StableDiffusionPrompt prompt) {
+        File pngFile = new File("pics/"
+                + prompt.getPrompt().split(" ")[(picMatchCounter == 1) ? picMatchCounter : (picMatchCounter = 0)] + "_"
```

Comment:
bug  here, doesnt add counter correcty as pic-file name, gotto generate a seed (PK) here for prompts